### PR TITLE
PR: erle pxfmini wiring

### PR DESCRIPTION
**Pull request by imuguruza**
Add to documentation pxf mini wiring page

### CODE REVIEW

File: common/source/docs/common-pxfmini-wiring-quick-start.rst [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+You could attach different dongle/devices to the USB ports that includes Raspberry Pi 2/3/Zero:
+
+
+WiFi dongle
+-----------
+
+Use a (regular or high gain) WiFi dongle to create a hotspot that can,
+for example, be used for sending telemetry data and commands to a Ground
+Station.  :ref: The images cooked`Connect ESCs and Motors <http://erlerobotics.com/docs/Artificial_Brains_and_Autopilots/Operating_System_images/Debian.html>` by  Erle-Robotics automatically generates a WiFi hotspot when the dongle is connected.
```
Comment (hamishwillee) [inline common/source/docs/common-pxfmini-wiring-quick-start.rst]: There is something broken about the ref in this line.

**Comment by hamishwillee** [comment]
There are a lot of images referenced that are not added by this PR. Are they added in another PR?

**Comment by imuguruza** [comment]
Yes, here #410

**Comment by hamishwillee** [comment]
@imuguruza Thanks - can you add the relevant images to each of their associated PRs, and make sure that they are lowest reasonable resolution? Otherwise we can't easily assess PRs like this one - and if we reject PRs we end up with orphaned images.

IN addition, please see the comment https://github.com/ArduPilot/ardupilot_wiki/pull/400#discussion_r60840555

**Comment by imuguruza** [comment]
OK, I'll fix that and make a new PR and reference to this

File: common/source/docs/common-pxfmini-wiring-quick-start.rst (hunk 1 above)
Comment (imuguruza) [inline common/source/docs/common-pxfmini-wiring-quick-start.rst]: I have changed that, hope is more clear now

**Comment by hamishwillee** [comment]
@imuguruza Thanks for this. In future, could you please try to rebase your changes on master rather than direct merging. I know this is not always possible and I personally sometimes have had trouble getting it to work. But it is much better/cleaner way to view the docs history, and also to understand precisely what you have changed vs what changes are an artifact of merging.

**Comment by imuguruza** [comment]
Yes, it's quite messy... I'll try to make it cleaner next time

**Pull request merged**